.env

```diff
@@ -0,0 +1,14 @@
+MYSQLDB_DATABASE=book_app_store
+
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=root
+
+MYSQL_ROOT_PASSWORD=12345678
+MYSQLDB_PASSWORD=12345678
+
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8081
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
the point of .env file is that it shouldn't be committed - it contains secrets. Not in your case, but that's a good practice to learn.